server/models.py

```diff
@@ -1643,3 +1644,93 @@ class CanvasAssignment(Model):
     @property
     def url(self):
         return '{}/assignments/{}'.format(self.canvas_course.url, self.external_id)
+
+#########
+# Files #
+#########
+
+class ExternalFile(Model):
+
+    id = db.Column(db.Integer, primary_key=True)
+    # Bucket/Folder that the file is stored in
+    container = db.Column(db.String(255), nullable=False)
+    filename = db.Column(db.String(1024), nullable=False)
+    object_name = db.Column(db.String(1024), nullable=False)
+
+    is_staff = db.Column(db.Boolean, nullable=False)
```

Comment:
Renaming is fine (I'm partial to `staff_file`). Good point about it making queries easier